aspnetcore/grpc/comparison.md

```diff
@@ -75,29 +77,28 @@ gRPC is well suited to the following scenarios:
 
-### Not Human Readable
+### Not human readable
 
 HTTP API requests are sent as text and can be read and created by humans.
 
 gRPC messages are encoded with Protobuf by default. While Protobuf is efficient to send and receive, its binary format is not human readable. Protobuf requires the message's interface description specified in the `*.proto` file to properly deserialize. Additional tooling needs to be used to analyze Protobuf payloads on the wire and to compose requests by hand.
-Features like [server reflection](https://github.com/grpc/grpc/blob/master/doc/server-reflection.md) and [gRPC command line tool](https://github.com/grpc/grpc/blob/master/doc/command_line_tool.md) exist to get around this limitation.
-Also, Protobuf messages support [conversion to and from JSON](https://developers.google.com/protocol-buffers/docs/proto3#json). The built-in JSON conversion provides a good way to convert the protobuf messages to/from human readable form when debugging.
+Features like [server reflection](https://github.com/grpc/grpc/blob/master/doc/server-reflection.md) and [gRPC command line tool](https://github.com/grpc/grpc/blob/master/doc/command_line_tool.md) exist to get around this limitation. Also, Protobuf messages support [conversion to and from JSON](https://developers.google.com/protocol-buffers/docs/proto3#json). The built-in JSON conversion provides a good way to convert the protobuf messages to/from human readable form when debugging.
 
-## Alternative Framework Scenarios
+## Alternative framework scenarios
 
 Other frameworks are recommended over gRPC in the following scenarios:
 
 * **Browser accessible APIs** - gRPC is not fully supported in the browser. gRPC-Web can offer browser support, but it has limitations and introduces a server proxy.
 * **Broadcast real-time communication** - gRPC supports real-time communication via streaming, but it does not have the concept broadcasting a message out to registered connections. For example, in a chat room scenario where new chat messages should be sent to all clients in the chat room, each gRPC call would need to individually stream new chat messages to the client. [SignalR](xref:signalr/introduction) is a good framework for this scenario. It has the concept of persistent connections and built-in support for broadcasting messages.
-* **Inter-process communication** - A process would need to host a HTTP/2 gRPC server to accept incoming calls. For Windows inter-process communication [named pipes with WCF](/dotnet/framework/wcf/feature-details/choosing-a-transport#when-to-use-the-named-pipe-transport) is a fast, lightweight method of communication.
+* **Inter-process communication** - A process would need to host a HTTP/2 server to accept incoming gRPC calls. For Windows inter-process communication [pipes](/dotnet/standard/io/pipe-operations) is a fast, lightweight method of communication.
```

Comment:
We're using the en dash for these types of lists. Instead of `-`, use `&ndash;` on Lines 99-101.

```suggestion
* **Inter-process communication** &ndash; A process must host an HTTP/2 server to accept incoming gRPC calls. For Windows, inter-process communication [pipes](/dotnet/standard/io/pipe-operations) is a fast, lightweight method of communication.
```